test/src/test/java/org/corfudb/integration/CorfuReplicationClusterConfigIT.java

```diff
@@ -1492,6 +1493,75 @@ public void testEnforceSnapshotSync() throws Exception {
         assertThat(oldShadowTs).isEqualTo(Long.parseLong(metadata.getVal()));
     }
 
+    @Test
+    public void testSnapshotApplyInterrupted() throws Exception {
+        // Write first batch of entries to active map
+        for (int i = 0; i < firstBatch; i++) {
+            try (TxnContext txn = activeCorfuStore.txn(NAMESPACE)) {
+                txn.putRecord(mapActive, StringKey.newBuilder().setKey(String.valueOf(i)).build(),
+                        IntValue.newBuilder().setValue(i).build(), null);
+                txn.commit();
+            }
+        }
+        assertThat(mapActive.count()).isEqualTo(firstBatch);
+        assertThat(mapStandby.count()).isZero();
+
+        int waitInSnapshotApplyMs = 10000;
+        activeReplicationServer = runReplicationServer(activeReplicationServerPort);
+        standbyReplicationServer = runReplicationServer(standbyReplicationServerPort, waitInSnapshotApplyMs);
+        log.info("Replication servers started...");
+
+        // Verify Sync Status
+        Sleep.sleepUninterruptibly(Duration.ofSeconds(3));
```

Comment:
no need of this initial sleep as we are already waiting till the sync status reaches the desired value.